### PR TITLE
Added extra configuration files.

### PR DESCRIPTION
**Pull request by MrFrontend**
- dnsmasq.conf and docker resolver
- script to start projects with Os X
- example docker-compose-osx.yml file for projects without this config file.
Updated readme

### CODE REVIEW

File: docker-compose-osx.yml
Comment (mkooij) [inline docker-compose-osx.yml]: please change to more generic hostname and add desciption :)

**Comment by jaapio** [comment]
Think it is wise to add a description how to setup things in the readme. Including the requirements.

File: docker-compose-osx.yml
Comment (MrFrontend) [reply docker-compose-osx.yml]: Agreed, gonna change it.

**Comment by MrFrontend** [comment]
Hello @jaapio,

I understand your concern, the explanation on how to set it up "step by step" is right now on the confluence page:
https://developers.polderknowledge.nl/confluence/display/COM/Docker+Dns+OSX

This documentation includes this project but also some extra setup.

We could duplicate the explanation here but it would not be really convenient for future maintainers as we loose the "single source of truth".

What we could do is copy from Confluence the setup regarding this project and leave the rest in Confluence.
What's your call on this @mkooij ?

**Comment by MrFrontend** [comment]
@jaapio , @mkooij  Can you review the documentation ?

**Review by jaapio — APPROVED** [review]

**Pull request merged**
